numpy/core/shape_base.pyi

```diff
@@ -45,18 +45,27 @@ def stack(
     arrays: Sequence[_ArrayLike[_SCT]],
     axis: SupportsIndex = ...,
     out: None = ...,
+    *,
+    dtype: None = ...,
```

Comment:
```suggestion
    dtype: DTypeLike = ...,
```
The same applies to the overloads below, note that it will have to be imported from the `numpy._typing` namespace first.

Reply:
Hello @BvB93 we changed the type of these keywords arguments, but 2 tests failed when we pushed to the GitHub (our local tests passed successfully). Do you have any tip to help us to solve this problem?